src/openai_cli/cli.py

```diff
@@ -14,23 +14,31 @@ def cli():
 @cli.command()
 @click.argument("source", type=click.File("rt", encoding="utf-8"))
 @click.option("-t", "--token", default="", help="OpenAI API token")
-def complete(source: io.TextIOWrapper, token: str) -> None:
+@click.option('-m', "--model", default="", help="OpenAI model option")
+def complete(source: io.TextIOWrapper, token: str, model: str) -> None:
     """Return OpenAI completion for a prompt from SOURCE."""
-    client = build_completion_client(get_token(token))
+    client = build_completion_client(token = get_token(token), model = get_model(model))
     prompt = source.read()
     result = client.generate_response(prompt)
     click.echo(result)
 
 
 @cli.command()
 @click.option("-t", "--token", default="", help="OpenAI API token")
-def repl(token: str) -> None:
+@click.option('-m', "--model", default="", help="OpenAI model option")
```

Comment:
text-davinci-003 is the most advanced model, I think it's to have it as a default here.